bpdv3/Assets/Scripts/Managers/TestUIManager.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace Assets.Scripts.Managers 
+{
+    using TMPro;
+    using UnityEngine;
+
+    public class TestUIManager : MonoBehaviour
+    {
+        #region Private & Const Variables
+
+        private float _timer;
+
+        #endregion
+
+        #region Public & Protected Variables
+
+        public TextMeshProUGUI TimerUiElement;
+
+        #endregion
+
+        #region Constructors
+
+        #endregion
+
+        #region Private Methods
+
+        private void Update()
+        {
+            IncrementTimer();
+        }
+
+        private void IncrementTimer()
+        {
+            _timer += Time.deltaTime;
+
+            if (TimerUiElement?.text == null) 
```

Comment:
Just a question so I can learn) the ? u have here, is this a ternary operator? I'm familiar with them but haven't seen it used like this before.:
TimerUiElement?.text